src/routes/list-store/+page.svelte

```diff
@@ -38,7 +38,7 @@
     <ul>
       {#each stores as { name, href }}
         <li>
-          <Links {href}>{name} &#8594;</Links>
+          <Links {href}><span class="truncate"> {name} </span> &#8594;</Links>
```

Comment:
what is it with the extra spaces ?

Reply:
I like that extra space, I know you don't haha